app/views/network_router/show.html.haml

```diff
@@ -0,0 +1,5 @@
+#main_div
+  - if %w(instances cloud_subnets).include?(@display) && @showtype != "compare"
+    = render :partial => "layouts/gtl", :locals => {:action_url => "show/#{@record.id}"}
```

Comment:
hm it's some partial url that is probably concatenated somewhere, so seems like next PR refactoring